Checking for files to update...
Files found. Executing script...
Created Success Directory
Created Error Directory
Running Upsert Patient
Data Loader requires Java JRE 17 or later. Checking if it is installed...
2024-04-09 10:27:13,620 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:310) - Initializing process engine
2024-04-09 10:27:13,620 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:325) - process.nameis specified in the command line. Loading DynaBean with id upsertPatient from process-conf.xml located in folder C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:13,635 INFO  [main] process.ProcessConfig getBeanFactory (ProcessConfig.java:106) - Loading process configuration from config file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\process-conf.xml
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
2024-04-09 10:27:14,887 INFO  [upsertPatient] config.Config createDir (Config.java:1496) - Config folder already exists: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:14,887 INFO  [upsertPatient] config.Config getInstance (Config.java:1529) - Looking for file in config path: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:14,887 INFO  [upsertPatient] config.Config getInstance (Config.java:1560) - User config is found in C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:14,902 INFO  [upsertPatient] config.LastRun load (LastRun.java:87) - Last run info will be saved in file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\upsert_lastRun.properties
2024-04-09 10:27:14,902 INFO  [upsertPatient] config.Config getInstance (Config.java:1567) - The controller config has been initialized
2024-04-09 10:27:14,902 INFO  [upsertPatient] process.ProcessRunner run (ProcessRunner.java:136) - Initializing process engine
2024-04-09 10:27:14,933 INFO  [upsertPatient] process.ProcessRunner run (ProcessRunner.java:156) - Logging in to: https://test.salesforce.com
2024-04-09 10:27:14,933 INFO  [upsertPatient] client.PartnerClient dologin (PartnerClient.java:626) - Beginning Partner Salesforce login ....
2024-04-09 10:27:14,980 INFO  [upsertPatient] client.PartnerClient loginInternal (PartnerClient.java:677) - Salesforce login to https://test.salesforce.com/services/Soap/u/60.0/ as user supersaml@wisewolves.com.wisewolves
2024-04-09 10:27:15,886 INFO  [upsertPatient] process.ProcessRunner run (ProcessRunner.java:159) - Setting field types
2024-04-09 10:27:17,095 INFO  [upsertPatient] process.ProcessRunner run (ProcessRunner.java:163) - Setting object reference types
2024-04-09 10:27:25,128 INFO  [upsertPatient] process.ProcessRunner run (ProcessRunner.java:167) - Creating Map
2024-04-09 10:27:25,128 INFO  [upsertPatient] dao.DataAccessObjectFactory getDaoInstance (DataAccessObjectFactory.java:55) - Instantiating data access object: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\Patients.csv of type: csvRead
2024-04-09 10:27:25,128 INFO  [upsertPatient] controller.Controller createDao (Controller.java:230) - Checking the data access object connection
2024-04-09 10:27:25,128 ERROR [upsertPatient] csv.CSVFileReader initalizeInput (CSVFileReader.java:291) - File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\Patients.csv not found.  Open failed.
java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\Patients.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
2024-04-09 10:27:25,142 FATAL [upsertPatient] controller.Controller createDao (Controller.java:233) - Error creating data access object
com.salesforce.dataloader.exception.DataAccessObjectInitializationException: File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\Patients.csv not found.  Open failed.
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:292) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
Caused by: java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\Patients.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) ~[dataloader-60.0.2.jar:?]
	... 8 more
2024-04-09 10:27:25,142 FATAL [main] process.ProcessRunner logErrorAndExitProcess (ProcessRunner.java:253) - Unable to run process
com.salesforce.dataloader.exception.MappingInitializationException: Error creating data access object
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:242) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
An error occurred during "Patient". Check DataLoader logs for details.
Running Upsert EncounterOrg
Data Loader requires Java JRE 17 or later. Checking if it is installed...
2024-04-09 10:27:28,286 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:310) - Initializing process engine
2024-04-09 10:27:28,286 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:325) - process.nameis specified in the command line. Loading DynaBean with id upsertEncounterOrg from process-conf.xml located in folder C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:28,286 INFO  [main] process.ProcessConfig getBeanFactory (ProcessConfig.java:106) - Loading process configuration from config file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\process-conf.xml
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
2024-04-09 10:27:29,281 INFO  [upsertEncounterOrg] config.Config createDir (Config.java:1496) - Config folder already exists: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:29,281 INFO  [upsertEncounterOrg] config.Config getInstance (Config.java:1529) - Looking for file in config path: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:29,281 INFO  [upsertEncounterOrg] config.Config getInstance (Config.java:1560) - User config is found in C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:29,281 INFO  [upsertEncounterOrg] config.LastRun load (LastRun.java:87) - Last run info will be saved in file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\upsert_lastRun.properties
2024-04-09 10:27:29,281 INFO  [upsertEncounterOrg] config.Config getInstance (Config.java:1567) - The controller config has been initialized
2024-04-09 10:27:29,294 INFO  [upsertEncounterOrg] process.ProcessRunner run (ProcessRunner.java:136) - Initializing process engine
2024-04-09 10:27:29,309 INFO  [upsertEncounterOrg] process.ProcessRunner run (ProcessRunner.java:156) - Logging in to: https://test.salesforce.com
2024-04-09 10:27:29,309 INFO  [upsertEncounterOrg] client.PartnerClient dologin (PartnerClient.java:626) - Beginning Partner Salesforce login ....
2024-04-09 10:27:29,341 INFO  [upsertEncounterOrg] client.PartnerClient loginInternal (PartnerClient.java:677) - Salesforce login to https://test.salesforce.com/services/Soap/u/60.0/ as user supersaml@wisewolves.com.wisewolves
2024-04-09 10:27:30,153 INFO  [upsertEncounterOrg] process.ProcessRunner run (ProcessRunner.java:159) - Setting field types
2024-04-09 10:27:31,349 INFO  [upsertEncounterOrg] process.ProcessRunner run (ProcessRunner.java:163) - Setting object reference types
2024-04-09 10:27:38,680 INFO  [upsertEncounterOrg] process.ProcessRunner run (ProcessRunner.java:167) - Creating Map
2024-04-09 10:27:38,680 INFO  [upsertEncounterOrg] dao.DataAccessObjectFactory getDaoInstance (DataAccessObjectFactory.java:55) - Instantiating data access object: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\EncounterOrgs.csv of type: csvRead
2024-04-09 10:27:38,680 INFO  [upsertEncounterOrg] controller.Controller createDao (Controller.java:230) - Checking the data access object connection
2024-04-09 10:27:38,680 ERROR [upsertEncounterOrg] csv.CSVFileReader initalizeInput (CSVFileReader.java:291) - File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\EncounterOrgs.csv not found.  Open failed.
java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\EncounterOrgs.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
2024-04-09 10:27:38,696 FATAL [upsertEncounterOrg] controller.Controller createDao (Controller.java:233) - Error creating data access object
com.salesforce.dataloader.exception.DataAccessObjectInitializationException: File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\EncounterOrgs.csv not found.  Open failed.
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:292) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
Caused by: java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\EncounterOrgs.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) ~[dataloader-60.0.2.jar:?]
	... 8 more
2024-04-09 10:27:38,696 FATAL [main] process.ProcessRunner logErrorAndExitProcess (ProcessRunner.java:253) - Unable to run process
com.salesforce.dataloader.exception.MappingInitializationException: Error creating data access object
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:242) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
An error occurred during "EncounterOrg". Check DataLoader logs for details.
Running Upsert ProviderContact
Data Loader requires Java JRE 17 or later. Checking if it is installed...
2024-04-09 10:27:42,476 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:310) - Initializing process engine
2024-04-09 10:27:42,476 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:325) - process.nameis specified in the command line. Loading DynaBean with id upsertProviderContact from process-conf.xml located in folder C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:42,476 INFO  [main] process.ProcessConfig getBeanFactory (ProcessConfig.java:106) - Loading process configuration from config file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\process-conf.xml
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
2024-04-09 10:27:43,569 INFO  [upsertProviderContact] config.Config createDir (Config.java:1496) - Config folder already exists: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:43,569 INFO  [upsertProviderContact] config.Config getInstance (Config.java:1529) - Looking for file in config path: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:43,569 INFO  [upsertProviderContact] config.Config getInstance (Config.java:1560) - User config is found in C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:43,569 INFO  [upsertProviderContact] config.LastRun load (LastRun.java:87) - Last run info will be saved in file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\upsert_lastRun.properties
2024-04-09 10:27:43,569 INFO  [upsertProviderContact] config.Config getInstance (Config.java:1567) - The controller config has been initialized
2024-04-09 10:27:43,584 INFO  [upsertProviderContact] process.ProcessRunner run (ProcessRunner.java:136) - Initializing process engine
2024-04-09 10:27:43,584 INFO  [upsertProviderContact] process.ProcessRunner run (ProcessRunner.java:156) - Logging in to: https://test.salesforce.com
2024-04-09 10:27:43,600 INFO  [upsertProviderContact] client.PartnerClient dologin (PartnerClient.java:626) - Beginning Partner Salesforce login ....
2024-04-09 10:27:43,618 INFO  [upsertProviderContact] client.PartnerClient loginInternal (PartnerClient.java:677) - Salesforce login to https://test.salesforce.com/services/Soap/u/60.0/ as user supersaml@wisewolves.com.wisewolves
2024-04-09 10:27:44,244 INFO  [upsertProviderContact] process.ProcessRunner run (ProcessRunner.java:159) - Setting field types
2024-04-09 10:27:45,316 INFO  [upsertProviderContact] process.ProcessRunner run (ProcessRunner.java:163) - Setting object reference types
2024-04-09 10:27:51,433 INFO  [upsertProviderContact] process.ProcessRunner run (ProcessRunner.java:167) - Creating Map
2024-04-09 10:27:51,433 INFO  [upsertProviderContact] dao.DataAccessObjectFactory getDaoInstance (DataAccessObjectFactory.java:55) - Instantiating data access object: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ProviderContacts.csv of type: csvRead
2024-04-09 10:27:51,449 INFO  [upsertProviderContact] controller.Controller createDao (Controller.java:230) - Checking the data access object connection
2024-04-09 10:27:51,449 ERROR [upsertProviderContact] csv.CSVFileReader initalizeInput (CSVFileReader.java:291) - File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ProviderContacts.csv not found.  Open failed.
java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ProviderContacts.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
2024-04-09 10:27:51,449 FATAL [upsertProviderContact] controller.Controller createDao (Controller.java:233) - Error creating data access object
com.salesforce.dataloader.exception.DataAccessObjectInitializationException: File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ProviderContacts.csv not found.  Open failed.
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:292) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
Caused by: java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ProviderContacts.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) ~[dataloader-60.0.2.jar:?]
	... 8 more
2024-04-09 10:27:51,464 FATAL [main] process.ProcessRunner logErrorAndExitProcess (ProcessRunner.java:253) - Unable to run process
com.salesforce.dataloader.exception.MappingInitializationException: Error creating data access object
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:242) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
An error occurred during "ProviderContact". Check DataLoader logs for details.
Running Upsert Clinical Encounter
Data Loader requires Java JRE 17 or later. Checking if it is installed...
2024-04-09 10:27:54,385 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:310) - Initializing process engine
2024-04-09 10:27:54,385 INFO  [main] process.ProcessRunner getInstance (ProcessRunner.java:325) - process.nameis specified in the command line. Loading DynaBean with id upsertClinicalEncounter from process-conf.xml located in folder C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:54,385 INFO  [main] process.ProcessConfig getBeanFactory (ProcessConfig.java:106) - Loading process configuration from config file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\process-conf.xml
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] config.Config createDir (Config.java:1496) - Config folder already exists: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] config.Config getInstance (Config.java:1529) - Looking for file in config path: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] config.Config getInstance (Config.java:1560) - User config is found in C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\config.properties
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] config.LastRun load (LastRun.java:87) - Last run info will be saved in file: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Conf\upsert_lastRun.properties
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] config.Config getInstance (Config.java:1567) - The controller config has been initialized
2024-04-09 10:27:55,502 INFO  [upsertClinicalEncounter] process.ProcessRunner run (ProcessRunner.java:136) - Initializing process engine
2024-04-09 10:27:55,516 INFO  [upsertClinicalEncounter] process.ProcessRunner run (ProcessRunner.java:156) - Logging in to: https://test.salesforce.com
2024-04-09 10:27:55,516 INFO  [upsertClinicalEncounter] client.PartnerClient dologin (PartnerClient.java:626) - Beginning Partner Salesforce login ....
2024-04-09 10:27:55,547 INFO  [upsertClinicalEncounter] client.PartnerClient loginInternal (PartnerClient.java:677) - Salesforce login to https://test.salesforce.com/services/Soap/u/60.0/ as user supersaml@wisewolves.com.wisewolves
2024-04-09 10:27:56,203 INFO  [upsertClinicalEncounter] process.ProcessRunner run (ProcessRunner.java:159) - Setting field types
2024-04-09 10:27:57,100 INFO  [upsertClinicalEncounter] process.ProcessRunner run (ProcessRunner.java:163) - Setting object reference types
2024-04-09 10:28:05,275 INFO  [upsertClinicalEncounter] process.ProcessRunner run (ProcessRunner.java:167) - Creating Map
2024-04-09 10:28:05,275 INFO  [upsertClinicalEncounter] dao.DataAccessObjectFactory getDaoInstance (DataAccessObjectFactory.java:55) - Instantiating data access object: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ClinicalEncounters.csv of type: csvRead
2024-04-09 10:28:05,290 INFO  [upsertClinicalEncounter] controller.Controller createDao (Controller.java:230) - Checking the data access object connection
2024-04-09 10:28:05,290 ERROR [upsertClinicalEncounter] csv.CSVFileReader initalizeInput (CSVFileReader.java:291) - File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ClinicalEncounters.csv not found.  Open failed.
java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ClinicalEncounters.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
2024-04-09 10:28:05,290 FATAL [upsertClinicalEncounter] controller.Controller createDao (Controller.java:233) - Error creating data access object
com.salesforce.dataloader.exception.DataAccessObjectInitializationException: File: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ClinicalEncounters.csv not found.  Open failed.
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:292) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.open(CSVFileReader.java:126) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.checkConnection(CSVFileReader.java:115) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.createDao(Controller.java:231) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:240) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
Caused by: java.io.FileNotFoundException: C:\Users\david\OneDrive\Documentos\WiseWolves\Bamboo\dataloader_v60.0.2\Bamboo\Data\ToUpdate\ClinicalEncounters.csv (The system cannot find the file specified)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:213) ~[?:?]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152) ~[?:?]
	at com.salesforce.dataloader.dao.csv.CSVFileReader.initalizeInput(CSVFileReader.java:265) ~[dataloader-60.0.2.jar:?]
	... 8 more
2024-04-09 10:28:05,290 FATAL [main] process.ProcessRunner logErrorAndExitProcess (ProcessRunner.java:253) - Unable to run process
com.salesforce.dataloader.exception.MappingInitializationException: Error creating data access object
	at com.salesforce.dataloader.controller.Controller.initializeOperation(Controller.java:242) ~[dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.run(ProcessRunner.java:168) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.ProcessRunner.runBatchMode(ProcessRunner.java:270) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.runApp(DataLoaderRunner.java:93) [dataloader-60.0.2.jar:?]
	at com.salesforce.dataloader.process.DataLoaderRunner.main(DataLoaderRunner.java:72) [dataloader-60.0.2.jar:?]
An error occurred during "ClinicalEncounter". Check DataLoader logs for details.
All processes completed successfully.
Moving and renaming files to the Updated folder...
        1 file(s) moved.
        1 file(s) moved.
        1 file(s) moved.
        1 file(s) moved.
